providers/prefab/pkg/provider_config.go

```diff
@@ -0,0 +1,12 @@
+package prefab
+
+type ProviderConfig struct {
+	Configs map[string]interface{}
+	APIKey  string
+	APIURLs []string
+	Sources []string
+	// EnvironmentNames             []string
+	// ProjectEnvID                 int64
+	// InitializationTimeoutSeconds float64
+	// OnInitializationFailure      OnInitializationFailure
```

Comment:
Should this be removed or is there a reason to keep it? Perhaps comment a TODO if so?